packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -41,6 +41,8 @@ export default class EditorPage {
 			this.accessibilityIdKey = 'contentDescription';
 			this.accessibilityIdSuffix = ', ';
 		}
+
+		driver.setImplicitWaitTimeout( 10000 );
```

Comment:
I see 🤔 By default this is 0, so setting this to 10s, makes sure each and every element query is "transformed" into its equivalent waitFor... neat!